stable/xray/README.md

```diff
@@ -334,6 +335,7 @@ The following table lists the configurable parameters of the xray chart and thei
 | `rabbitmq-ha.replicaCount`                     | RabbitMQ Number of replica                   | `1`                  |
 | `rabbitmq-ha.rabbitmqUsername`                 | RabbitMQ application username                | `guest`              |
 | `rabbitmq-ha.rabbitmqPassword`                 | RabbitMQ application password                | ` `                  |
+| `rabbitmq-ha.existingSecret`                   | RabbitMQ existingSecret                      | ` `                  |
```

Comment:
Why are they different? It looks odd to me

Reply:
Hmm, upstream charts have different existingSecret names 
https://github.com/bitnami/charts/blob/master/bitnami/rabbitmq/values.yaml#L75
https://github.com/kubernetes/charts/blob/master/stable/rabbitmq-ha/README.md

Reply:
mm okI see,